chapters/beam_loader.asciidoc

```diff
@@ -49,3 +49,231 @@ e.g. +erts/emulator/x86_64-unknown-linux-gnu/opt/smp/+).
 
 The same program (beam_makeops) also generates the Erlang code for the
 compiler back end +beam_opcodes.erl+.
+
+=== Understanding ops.tab
+
+// Missing description about build specific options, i.e. the %unless construct
```

Comment:
Where can I see an example of this used? I just followed the style in this file, i.e. the comment at the top of the file.